PENDING.md

```diff
@@ -3,11 +3,13 @@
 BREAKING CHANGES
 
 * Gaia REST API (`gaiacli advanced rest-server`)
+  * [gaia-lite] [\#2182] Renamed and merged all redelegations endpoints into `/stake/redelegations`
```

Comment:
Looks like some of these additions don't belong here...possible from a bad merge/rebase.

Reply:
yeah it was from sunny's PR which had `PENDING` entries from other releases. Also that PR had breaking changes which weren't added (that's why I changed them here). 

Already talked with @jackzampolin and he said it was fine to do this changes on the `PENDING` here